guides/source/active_storage_overview.md

```diff
@@ -539,60 +539,76 @@ Analysis requires the `mini_magick` gem. Video analysis also requires the [FFmpe
 
 [`analyzed?`]: https://api.rubyonrails.org/classes/ActiveStorage/Blob/Analyzable.html#method-i-analyzed-3F
 
-Transforming Images
--------------------
+Displaying Images, Videos, and PDFs
 
-To enable variants, add the `image_processing` gem to your `Gemfile`:
+Active Storage supports [representing][] a variety of files. You can use the `representation` method on an attachment to display an image variant, or a preview of a video or PDF. Before calling `representation`, check if the attachment can be represented by calling `representable?`. Some file formats can't be previewed natively in the browser (eg. Word documents); if `representable?` returns false you may want to [link to](#linking-to-files) the file instead.
```

Comment:
* *Most* of this guide seems to be wrapped at 80 characters, so this should be wrapped too, for consistency.
* The first occurrences of `representation` and `representable?` should be linked to the API docs, a la #40706.
* Reference-style links are document-wide, meaning if another `[representing]: ` definition is added elsewhere, the last definition will win for all `[representing][]` links.  So reference identifiers should be uniquely associated with their URL.  Since we will already be linking to `Representable#representation` and `Representable#representable?`, I would just forgo linking `[representing][]`.
* It's not so much that some formats "can't be previewed natively in the browser."  Rather, Active Storage cannot generate a preview for some formats, out-of-the-box.